partner_communication/static/src/less/communication_dashboard.less

```diff
@@ -0,0 +1,223 @@
+.o_kanban_view.o_kanban_dashboard.o_partner_communication_kanban {
+    @communication-table-spacing: 10px;
+
+    .o_kanban_record {
```

Comment:
This class doesn't need to be redefined, no? I assume you can use it from the core modules without to have to define it again, unless you did some changes?